CMSgov/mint-app
# [EASI-2408] Truncating the Discussions Panel

# EASI-2408

## Changes and Description

- Moved out `FormatDiscussions` and `discussionComponent` into its own component
- Utilized `isAccordionExpanded` state to handle truncating the discussions. (
  - if `isAccordionExpanded === false` , then `discussionsContent.slice(0,5)`, otherwise render the entire discussionsContent
- Add `View more questions` in the list of discussions in Discussion component




## How to test this change

1. Navigate to any model plan and add 6 questions in the Discussion modal
2. Ensure `View more questions` button is only visible if there is more than 5 questions
![image](https://user-images.githubusercontent.com/2349702/201759168-0e8faedf-a89a-4f82-9e57-734f30ef5bb6.png)

3. Click link and ensure that the rest of the questions show up and that the link will disappear
4. Refresh to see the discussion panels truncated again and view more button reappears



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
